### ISSUE TITLE
If this plugin support ClientID Authentication?

### ISSUE THREAD

**Issue by defshine** [emqx/emqx-auth-mongo#12]
The  emqttd broker provides Username, ClientId, LDAP and anonymous authentication modules by default, I can enable many modules ,and broker will process the modules chain.

I want to know if this plugin can support Username and ClientId at the same time?

**Comment by huangdan** [comment]
hi @defshine 
Sorry for my late reply!
The emq_auth_http plugin supports both Username and ClientId certification
Tips:https://github.com/emqtt/emq_auth_http.git

**Comment by anhldbk** [comment]
@emqplus Hey Feng, the new version supports this feature, right? I see the configuration in `mqtt_acl`:
```js
{
    username: "username",
    clientid: "clientid",
    publish: ["topic1", "topic2", ...],
    subscribe: ["subtop1", "subtop2", ...],
    pubsub: ["topic/#", "topic1", ...]
}
```

**Comment by ig0r** [comment]
guys, please implement it as optional

**Comment by huangdan** [comment]
@defshine, Has been fixed and released at v2.3.11.
https://github.com/emqx/emqx-auth-mongo/pull/122

**Comment by huangdan** [comment]
@defshine, I will close this issue, if there is a problem, will reopen it.
